rasa/nlu/convert.py

```diff
@@ -23,7 +26,8 @@ def convert_training_data(
         output = td.as_markdown()
     else:
         print_error(
-            "Did not recognize data format. Supported data formats: 'json' and 'md'."
+            "Did not recognize data format. Supported data formats: 'json' and 'md'. "
+            "Specify the desired data format with '--format'."
```

Comment:
I thought we got rid of --format in convert?

Reply:
if so, all this logic can go, but supported data formats should go in the "supply a valid nlu file" message

Reply:
We did not. Tom pointed out that you there are not just json and markdown data formats. NLU data can also be in WIT, LUIS, or Dialogflow data format. So we kept the `--format` argument.

Reply:
Ah gotcha, okay. Then I would say "specify the desired output format" or something of the sort